### DISCUSSION TITLE
save session (all variable, funcs etc)?

### DISCUSSION THREAD

**Discussion by mralusw** [fish-shell/fish-shell#10423] [General]
Is there a way to save / export the entire current state of the fish shell (including variables, abbreviations, functions) to a "session file" and later load that session file in another shell process?

**Comment by zanchey** [comment]
No, not at present.

**Comment by henri** [comment]
It is possible to use the following commands in order to gather the information you would want to save.

- ```alias``` 
- ```functions```
- ```env```

With the output from those commands you could then be even more specific about what you would like to save / restore.
